### PR TITLE
i-bem__dom: fix findBlock[s]Outside behaviour (close #845)

### LINKED ISSUE #845: i-bem__dom: findBlocksOutside adds self to found results

### PR DESCRIPTION
**Pull request by aristov**
Closes #845

### CODE REVIEW

File: common.blocks/i-bem/__dom/i-bem__dom.js [hunk 1]
```diff
@@ -355,7 +355,7 @@ DOM = BEM.decl('i-bem__dom',/** @lends BEMDOM.prototype */{
                     buildClass(blockName) :
                     buildClass(blockName, block.modName, block.modVal)) +
                 (onlyFirst? ':first' : ''),
-            domElems = ctxElem.filter(selector);
+            domElems = select === 'parents'? $() : ctxElem.filter(selector);
```
Comment (aristov) [inline common.blocks/i-bem/__dom/i-bem__dom.js]: TODO: omit `$()`.

**Comment by tadatuta** [comment]
should this task be included into 2.7 milestone?

**Comment by narqo** [comment]
I think it's not that important, so it can be scheduled to any future bug fix version.

**Comment by aristov** [comment]
Done.

**Comment by aristov** [comment]
Done.

**Comment by narqo** [comment]
Looks the original issue breaks backward compatibility. After discussion with @dfilatov decided to keep current behaviour as is.

**Pull request closed without merging**